### PR TITLE
Editing + Visualizations for JavaScript chapter

### PR DESCRIPTION
**Pull request by obto**
Progress on #218 and #237

### CODE REVIEW

File: src/content/en/2019/javascript.md [hunk 1]
```diff
@@ -7,215 +7,182 @@ reviewers: [rviscomi, obto, paulcalvano, mathiasbynens]
...
-when JavaScript is processed by a web page.
+JavaScript is the most costly resource we send to browsers, having to be downloaded, parsed, compiled and then finally executed. Although browsers have significantly decreased the time it takes to parse and compile scripts, [download and execution have become the most expensive stages](https://v8.dev/blog/cost-of-javascript-2019) when JavaScript is processed by a web page.
 
-Sending smaller JavaScript bundles to the browser is the best way to reduce download times, and in
-turn improve page performance. **But how much JavaScript do we really use?**
+Sending smaller JavaScript bundles to the browser is the best way to reduce download times, and in turn improve page performance. **But how much JavaScript do we really use?**
 
 Here's a chart showing the amount of bytes shipped for different percentiles:
 
 ```<insert graphic of metric 01_01a>```
```
Comment (rviscomi) [inline src/content/en/2019/javascript.md]: Remove all of these extra placeholders

File: src/content/en/2019/javascript.md [hunk 2]
```diff
@@ -7,215 +7,182 @@ reviewers: [rviscomi, obto, paulcalvano, mathiasbynens]
...
+<figure>
+	<iframe aria-labelledby="fig1-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1974602890&format=interactive"></iframe>
```
Comment (rviscomi) [inline src/content/en/2019/javascript.md]: For each iframe we should also have an img equivalent for mobile. See #237 ("interactive figures")

**Comment by obto** [comment]
@rviscomi I finished adding the visualizations but was only able to give a quick run through the content in terms of copy-editing. 

Sadly I have to call it here 😞 , as I got a flight to catch in about 10 hours.

**Comment by rviscomi** [comment]
No worries. I'll merge and give it another pass.

**Pull request merged**